api.Dockerfile

```diff
@@ -27,9 +31,13 @@ RUN npm install
 
 COPY ./devU-api .
 
-COPY --from=config /stage/default.yml ./config/default.yml
+COPY --from=config-builder /config/default.yml ./config/default.yml
+
 COPY --from=module_builder /tmp/devu-shared-modules ./devu-shared-modules
 
+# Indicate that the api is running in docker; value here is irrelevant
+ENV dev=0
```

Comment:
Let's choose a better variable name than `dev` for this. I see `isDocker` below, which sounds better.

Reply:
how about ```is_docker``` then